app/containers/MessageBox/index.js

```diff
@@ -111,7 +149,9 @@ class MessageBox extends Component {
 			},
 			commandPreview: [],
 			showCommandPreview: false,
-			command: {}
+			command: {},
+			fullScreenUp: false,
```

Comment:
Perhaps we could have a slightly more boolean name for `fullScreenUp` e.g. `isFullScreenUp` or even just `isFullScreen` or `isOpen`.

Reply:
Yep. `isFullscreen` is better.